sample-windows.ini

```diff
@@ -0,0 +1,45 @@
+[General]
+
+hostname = 0.0.0.0
+
+portNumber = 8000
+
+; If any symbols of interest aren't available locally (e.g. Windows DLLs), uncomment line below
+
+remoteSymbolServer = http://symbolapi.mozilla.org:80/
+
+
+
+; These options are unlikely to need editing by users
+
+maxCacheEntries = 2000000
+
+mruSymbolStateFile = %(TEMP)s\snappy-mru-symbols.json
+maxMRUSymbolsPersist = 10
+
+[Log]
+
+maxFiles = 10
+
+maxFileSize = 1048576
+
+logPath = %(TEMP)s\log\snappy
```

Comment:
do we create the log directory ourselves? My Windows machine doesn't have %TEMP%\log\